src/parser.rs

```diff
@@ -24,3 +26,32 @@ pub fn parse_file(path_str: &str) -> Result<Stylesheet, String> {
     }
   }
 }
+
+/// Load an XSLT stylesheet from string UTF-8 in byte format
+pub fn parse_string(file_string: Vec<u8>, url: &str) -> Result<Stylesheet, String> {
```

Comment:
any reason `parse_string` doesn't take a String argument (or possibly `&str`), but a low-level vector of bytes? That may fit better with a `parse_bytes` method.

Reply:
Makes sense, renamed the method ✅ 
I think the string most likely is still taken from the file (like in the test) or received via network anyway (my use case), both of which uses byte representation.

Anyhow, for `&str` should be possible to do `.as_bytes()` before passing it to the function :)